### DISCUSSION TITLE
getSubscriptions doesn't return price attribute in Subscription object on Android

### DISCUSSION THREAD

**Discussion by karel-suchomel-ed** [hyochan/react-native-iap#2033] [Q&A]
After I updated the package to the latest version, the android subscriptions, that I receive from getSubscriptions(), doesn't include price and currency attribute. Am I supposed to dig it up from subscriptionOfferDetails attribute, or is this only development issue?
I tried running the app in release mode, and the issue was still there.

**Comment by Hibbem** [comment]
After upgrading my version, I now have the same problem.. Did you manage to fix it? Now we have to cast it and check which platform we have.. It used to be too easy I guess? 😅

**Reply by karel-suchomel-ed** [reply]
I had to dig up the price from the _subscriptionOfferDetails_ attribute and split the subscription logic for ios and android

**Comment by KarimScript** [comment]
I get the price for a subscription like this : item.subscriptionOfferDetails[0].pricingPhases.pricingPhaseList[0].formattedPrice
